jiant/shared/model_resolution.py

```diff
@@ -223,5 +283,23 @@ def resolve_tokenizer_class(model_type):
 def resolve_is_lower_case(tokenizer):
     if isinstance(tokenizer, (transformers.BertTokenizer, transformers.AlbertTokenizer)):
         return tokenizer.basic_tokenizer.do_lower_case
+    elif isinstance(tokenizer, transformers.XLMTokenizer):
+        return tokenizer.do_lowercase_and_remove_accent
     else:
         return False
+
+
+def bart_or_mbart_model_heuristic(model_config: transformers.BartConfig) -> ModelArchitectures:
+    # assert model_config.scale_embedding == \
+    #     model_config.normalize_before == \
+    #     model_config.add_final_layer_norm
+    if model_config.scale_embedding:
+        model_arch = ModelArchitectures.MBART
+    else:
+        model_arch = ModelArchitectures.BART
+    warnings.warn(
+        "No clean way of disambiguating between BART and mBART models based on"
+        " the transformer class, so we're falling back on a heuristic. "
+        f"Guessing: {model_arch}"
```

Comment:
`is_valid_mbart()`? https://huggingface.co/transformers/_modules/transformers/configuration_bart.html#BartConfig.is_valid_mbart

Reply:
True, I can switch over to using that.